tests/core/tests/test_widgets.py

```diff
@@ -62,14 +63,11 @@ def test_clean(self):
         self.assertEqual(self.widget.clean("13.08.2012 18:00:00"),
                          self.datetime)
 
-    @override_settings(USE_TZ=True)
+    @override_settings(USE_TZ=True, TIME_ZONE='Europe/Ljubljana')
```

Comment:
What do you think about keeping a test for when the timezone is not set in the settings? Unless I'm misunderstanding something, the package should be able to handle both cases?

Reply:
When timezone is not specified Django use s default one (not remember which one it is). So it is always there, and specifying is more explicit